glastopf/glastopf.py

```diff
@@ -70,8 +70,15 @@ def __init__(self, config="glastopf.cfg", work_dir=os.getcwd()):
             "gid": conf_parser.get("webserver", "gid").encode('latin1'),
             "proxy_enabled": conf_parser.get("webserver", "proxy_enabled").encode('latin1'),
             "banner": conf_parser.get("misc", "banner").encode('latin1'),
+            "sensorid": conf_parser.get("sensor", "sensorid").encode('latin1'),
         }
 
+        if self.options['sensorid'] == "None":
+            self.options['sensorid'] = str(uuid.uuid4())
+            conf_parser.set('sensor', 'sensorid', self.options['sensorid'])
+            with open((os.path.join(self.work_dir,config)), 'wb') as configfile:
```

Comment:
Have you seen this: http://www.voidspace.org.uk/python/configobj.html#writing-a-config-file